modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/IgniteSnapshotManager.java

```diff
@@ -361,19 +356,19 @@ public static String partDeltaFileName(int partId) {
         MetricRegistry mreg = cctx.kernalContext().metric().registry(SNAPSHOT_METRICS);
 
         mreg.register("LastSnapshotStartTime", () -> lastSeenSnpFut.startTime,
-            "The system time approximated by 10 ms when the last cluster snapshot operation has been started.");
+            "The system time approximated by 10 ms of the last started cluster snapshot request on this node.");
         mreg.register("LastSnapshotEndTime", () -> lastSeenSnpFut.endTime,
-            "The system time approximated by 10 ms when the last cluster snapshot operation has been finished.");
+            "The system time approximated by 10 ms of the last started cluster snapshot request on this node.");
```

Comment:
The description is the same now. "approximated by 10 ms" can be omitted, but it's up to you.

Reply:
Fixed.